src/ontology/mondo-ingest.Makefile

```diff
@@ -575,6 +575,20 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+# update-repo-wrapper
+UPDATE_REPO_ODK_TAG := latest
+ifdef ODK_TAG
+	UPDATE_REPO_ODK_TAG := $(ODK_TAG)
+endif
+
+.PHONY: update-repo-wrapper
+update-repo-wrapper:
```

Comment:
## `update-repo-wrapper`
1. Temporarily updates `run.sh.conf` `ODK_TAG` to value of `UPDATE_REPO_ODK_TAG`
2. Runs `update_repo` twice
- We can change this if ever https://github.com/INCATools/ontology-development-kit/issues/989 gets fixed.